Feature/446 gridlines

charles-b-stb opened the pull request:
## Issue
- close #446

## Overview (Required)
- Simply adding dotted gridlines to the grid style timetable view. 

Draft: The vertical lines are not yet working and I am looking for advice.
Branched off of the [Grid UI pull request](https://github.com/DroidKaigi/conference-app-2024/pull/367) so that will need to be merged first.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
<img src="" width="300" /> | <img src="https://github.com/user-attachments/assets/9bb9e40f-8bcc-4fbf-96f0-3194d370ad96" width="300" />

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

charles-b-stb commented:
Let's try just adding horizontal graph lines for now. 

If anyone can think of a reasonable way to add vertical lines I would appreciate it. 🙇

MrSmart00 commented on app-ios/Sources/TimetableFeature/TimetableListView.swift:
```diff
@@ -192,6 +194,39 @@ struct TimeGroupMiniList: View {
+        } else {
+            shape.frame(width: 1).padding(0)
+        }
+        //shape
```
What's this comment🤔?

charles-b-stb replied:
Removed

MrSmart00 commented on app-ios/Sources/TimetableFeature/TimetableListView.swift:
```diff
@@ -192,6 +194,39 @@ struct TimeGroupMiniList: View {
+}
+
+struct LineShape: Shape {
+    public var axis: Axis
```
this property is 'let' more than better👍

MrSmart00 commented on app-ios/Sources/TimetableFeature/TimetableListView.swift:
```diff
@@ -192,6 +194,39 @@ struct TimeGroupMiniList: View {
 }
 
+struct DashedDivider: View {
+    public var axis: Axis
```
this property is 'let' more than better👍

MrSmart00 replied:
you should change here too🙏

MrSmart00 reviewed (commented):
THX!!
I left comments!
please check them🙏

charles-b-stb commented:
@MrSmart00 Responded to comments, thanks!

charles-b-stb commented:
@MrSmart00 Sorry I missed that last time (it looked too similar to the other item I corrected and I did not notice they were separate the first time I looked at it.)

Fixed now.

MrSmart00 reviewed (approved):
Thanks your commits!
LGTM👍
